.github/workflows/build.yml

```
@@ -66,15 +66,101 @@ jobs:
+              echo "Pull request event. Vulnerabilities found, but continuing (non-blocking)."
+            fi
+          else
+            echo "No HIGH/CRITICAL vulnerabilities found. Safe to proceed."
+          fi
+                
+      - name: Push Image
+        uses: docker/build-push-action@v6
+        with:
+          context: .
+          build-args: VERSION=v0.0.0-rolling+${{ github.sha }}
+          tags: ${{ steps.metadata.outputs.tags }}
```

actionlint (1.7.8)

property "metadata" is not defined in object type {login-ecr: {conclusion: string; outcome: string; outputs: {string => string}}; meta: {conclusion: string; outcome: string; outputs: {string => string}}; trivy_amd64: {conclusion: string; outcome: string; outputs: {string => string}}}

(expression)